sway/input/seatop_default.c

```diff
@@ -73,6 +73,9 @@ static enum wlr_edges find_edge(struct sway_container *cont,
 			cont->border == B_CSD) {
```

Comment:
`cont->border == B_NORMAL` during the reproduction of this bug, so this branch is never triggered.